.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Feature request
+about: Suggest an idea for GPflow
+labels: enhancement
+---
+
+<!-- Lines like this are comments and will be invisible -->
+
+# Feature request
+
+<!-- Please give a clear and concise description of your feature proposal! -->
```

Comment:
You've used a different sentence structure in various places where you ask a user to add information. I think it would be clearer if you used the same structure everywhere, such as
```
<!-- Add X here -->
```
or
```
<!-- Insert X here -->
```

Reply:
I've just followed GitHub's issue template templates..